src/screens/Dashboard/DashboardScreen.tsx

```diff
@@ -59,33 +59,27 @@ class DashboardScreen extends Component<Props, State> {
       .catch(() => {
         this.props.navigation.navigate(Route.CreatePin);
       });
-    this.props.loadWallets();
+    await this.props.loadWallets();
```

Comment:
if you have no logic after await, there's no difference in the way how it works ;)